doc/source/reference/c-api.iterator.rst

```diff
@@ -603,15 +602,18 @@ Construction and Destruction
             Indicates that only elements which the operand with
             the ARRAYMASK flag indicates are intended to be modified
             by the iteration. In general, the iterator does not enforce
-            this, it is up to the code doing the iteration to follow
-            that promise. Code can use the :c:func:`NpyMask_IsExposed`
-            inline function to test whether the mask at a particular
-            element allows writing.
+            this, it is up to the code doing the iteration to follow that
+            promise.
+
+            Essentially, this array is the mask for all `writemasked <numpy.nditer>` operands.
+            Code uses the ``writemasked`` flag which indicates that only
+            elements where the chosen arraymask operand is True will
+            be written to.
```

Comment:
text should be limited to 80 columns, you can break on any space even if it is inside a reference